CONTRIBUTING.md

```diff
@@ -18,9 +18,9 @@ propose changes to this document in a pull request.
 but before doing that please read the notes below and include as many details as
 possible with your report. If you can, please include:
   * The version of Electron you are using
-  * The operating system you are using
-  * If applicable, what you were doing when the issue arose and what you
-  expected to happen
+  * The specifications of the operating system you are using
```

Comment:
Asking for Operating system would not be sufficient I felt. We need to know the specifications of it to understand the issue more concisely. I might be wrong, but that is what I was thinking.